src/components/SinglePastCommitInfo.tsx

```diff
@@ -65,10 +59,31 @@ export class SinglePastCommitInfo extends React.Component<
     super(props);
     this.state = {
       displayDelete: false,
-      displayReset: false
+      displayReset: false,
+      info: "",
+      filesChanged: "",
+      insertionCount: "",
+      deletionCount: "",
+      list: [],
+    
     };
+    this.showPastCommitWork();
   }
 
+  showPastCommitWork = async () => {
+    let gitApi = new Git();
+    let detailedLogData = await gitApi.detailedLog(this.props.data.commit, this.props.topRepoPath);
```

Comment:
How do you feel about handling the error callback from this API? I'd like to not fail silently, at-least.